### DISCUSSION TITLE
Global Drafts

### DISCUSSION THREAD

**Discussion by esethna** [mattermost/community#63] [Practical Ideas]
I want to see all drafts for parent posts and replies in one place across all teams. This becomes even more important with collapsed reply threads!

![image](https://user-images.githubusercontent.com/14333569/123145973-3fcf9280-d412-11eb-94d9-f4c5000172c4.png)

**Comment by bbodenmiller** [comment]
Global meaning across all teams. Originally posted at https://github.com/mattermost/community/discussions/43#discussioncomment-907644

**Reply by esethna** [reply]
Updated the description

**Comment by Willyfrog** [comment]
I'd love this, specially with a number of recent drafts (drafts not sent in the last X time)

This would help me realize that the post I thought I have sent was not and is waiting for me to hit the send butron

**Reply by esethna** [reply]
Ya we could have smart notifications around this, for example, you have a draft older than x days

**Comment by chetanyakan** [comment]
Implementation can be like Twitter, where drafts are stored in the local storage of each browser/mobile device rather than in the server.

**Reply by Willyfrog** [reply]
i'd rather have this in the server, that way I can start a draft in the mobile while heading home and do a final review on the computer with a bigger screen

**Reply by esethna** [reply]
Agree with the server, my biggest use case for this is moving between mobile and web. We already store drafts in local storage don't we?

**Comment by Willyfrog** [comment]
Also, could this be also be used in a kind of "templating" messages? having the draft being permanent and just use it in a channel as a way to quickly compose?

**Comment by chetanyakan** [comment]
On a side note, this implementation can also be done via a plugin.
Using the LHS header component with some modal that allows selecting a draft message and the team and channel to post that message to.

**Reply by esethna** [reply]
Could be plugin, but I'd like to see this as part of the core product when you think about how common the use case is for drafts
